.circleci/config.yml

```diff
@@ -33,14 +34,22 @@ jobs:
     <<: *defaults
     docker:
     - image: circleci/python:3.4
+    - image: mongo:3.2.19
   test-3.5:
     <<: *defaults
     docker:
     - image: circleci/python:3.5
+    - image: mongo:3.2.19
```

Comment:
How was this working before w/o mongo running in the container? Was it silently skipping mongo tests?

Reply:
Yep, no mongo tests were run on py3.